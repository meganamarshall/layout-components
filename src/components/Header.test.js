import React from 'react';
import { shallow } from 'enzyme';

describe('Header component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Header />);
  })
})
