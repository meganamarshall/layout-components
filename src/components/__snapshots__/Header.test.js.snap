// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches header snapshot 1`] = `
<header>
  <h1>
    My Color
  </h1>
</header>
`;
